doc/api/fs.md

```diff
@@ -1071,6 +1071,16 @@ including subdirectories and files.
 When copying a directory to another directory, globs are not supported and
 behavior is similar to `cp dir1/ dir2/`.
 
+### `fsPromises.exists(path)`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* `path` {string|URL} The filepath to check.
+
+Checks whether the given filepath exists.
```

Comment:
That's not correct, the current implementation checks if the current process can access a file at that path.